### PR TITLE
fixes test/unit/jobs/test_job_output_checker.py; strange default?

### PR DESCRIPTION
**Pull request by caleb-easterly**
Unclear on the implications of this. 

In `lib/galaxy/tools/parser/interface.py`, the default for `ToolStdioRegex.error_level` was `"fatal"`. The following test passes under Python2 but fails under Python3: 

https://github.com/galaxyproject/galaxy/blob/01ba96a7acda5f321e3f3f0975909615122a7880/test/unit/jobs/test_job_output_checker.py#L62-L68

This is because, in the following

https://github.com/galaxyproject/galaxy/blob/01ba96a7acda5f321e3f3f0975909615122a7880/lib/galaxy/jobs/error_level.py#L21-L26

`'fatal' > 0 and 'fatal' <= StdioErrorLevel.MAX` is `false` in Python 2 but throws an error in Python 3. It seems like these shouldn't be compared at all...but I might not have fixed it in the right way.

xref. #1715

### CODE REVIEW

**Comment by martenson** [comment]
@galaxybot test this

**Review by mvdbeek — APPROVED** [review]
Thanks @caleb-easterly!

**Comment by dannon** [comment]
This is the correct fix, but we should probably use the constant `StdioErrorLevel.FATAL` (which is 3).  Want to push one more changeset, or should I?  (and you can drop the TODO as well, since it's suggesting to use the constant here)

**Comment by caleb-easterly** [comment]
I probably wouldn't get to it for an hour or so - feel free to make the necessary changes if you're looking at it now!

**Comment by dannon** [comment]
@caleb-easterly Sure, I'll do it now, thanks!

File: lib/galaxy/tools/parser/interface.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 import six
 
+from galaxy.jobs.error_level import StdioErrorLevel
```
Comment (mvdbeek) [inline lib/galaxy/tools/parser/interface.py]: I was discussing that with @jmchilton yesterday, and we didn't want to import from galaxy.jobs to keep this separate. We could move the error_level code into tools though ?

Comment (dannon) [reply lib/galaxy/tools/parser/interface.py]: @mvdbeek Sure, that sounds good to me -- happy to move it wherever, I just want to use the constant.

Comment (jmchilton) [reply lib/galaxy/tools/parser/interface.py]: Pushed a fix for this, the refactoring I was hoping for actually eliminates some need for delayed imports and such so we closer to untangling circular imports in Galaxy I think. Thanks for working on this all!

**Comment by nsoranzo** [comment]
I've rebased and updated @jmchilton's last commit to also update the import in `test/unit/jobs/test_job_output_checker.py`

**Review by nsoranzo — APPROVED** [review]

**Pull request merged**